src/app/shared/markdown-wrapper/markdown-wrapper.service.ts

```diff
@@ -87,7 +87,9 @@ export class MarkdownWrapperService {
    */
   customCompile(data, baseUrl): string {
     const parseOptions = { markedOptions: { baseUrl: baseUrl } };
-    return this.markdownService.parse(data, parseOptions);
```

Comment:
While it is good that you can reproduce (and fix)!

I'm always a little wary of regular expressions. 
e.g. could some other content accidentally get modified or could tab characters get left in accidentally

Wondering if you tried the `gfm` option to see if it matches github's behaviour 

I also see some changes  in how tabs/tables are handled in v4.0.14 through v4.0.11 so you could try updating past them to see if that changes anything 
https://github.com/markedjs/marked/releases/tag/v4.0.14
https://github.com/markedjs/marked/releases/tag/v4.0.12
https://github.com/markedjs/marked/releases/tag/v4.0.11

Reply:
Setting `gfm: true` didn't fix the table formatting, and I also tried updating the version (we were already using `marked@4.2.3` so I tried v4.3.0) but this unfortunately doesn't resolve the issue either.